Source/KSCrash/Recording/KSCrash.m

```diff
@@ -141,24 +141,25 @@ + (instancetype) sharedInstance
     return sharedInstance;
 }
 
-+ (instancetype) sharedInstanceWithBasePath:(NSString *)basePath
++ (instancetype) sharedInstanceWithBasePath:(NSString *)basePath andBundleName:(NSString *)bundleName
```

Comment:
If the goal of this change is to allow loading crashes from the extension app in the host app then this approach will not work. I recommend exposing a "static" function in KSCrash instance that directly access the internal "loading" of the crash report functions instead.